packages/odyssey-design-tokens/src/font.json

```diff
@@ -28,8 +44,14 @@
       "bold": { "value": "600" }
     },
     "lineHeight": {
-      "base": { "value": 1.5 },
-      "heading": { "value": 1.3 }
+      "body": { "value": 1.42857143 },
```

Comment:
These numbers are yucky in order to get 14px and 16px to share common baselines.